tt.go

```diff
@@ -0,0 +1,274 @@
+	TypeBoolean   TypeName = "boolean"
+	TypeString    TypeName = "string"
+	TypeInteger   TypeName = "integer"
+	TypeUnsigned  TypeName = "unsigned"
+	TypeDouble    TypeName = "double"
+	TypeNumber    TypeName = "number"
+	TypeDecimal   TypeName = "decimal"
+	TypeDatetime  TypeName = "datetime"
+	TypeUUID      TypeName = "uuid"
+	TypeArray     TypeName = "array"
+	TypeMap       TypeName = "map"
+	TypeVarbinary TypeName = "varbinary"
+	TypeScalar    TypeName = "scalar"
+	TypeAny       TypeName = "any"
+	TypeInterval  TypeName = "interval"
+)
+
+const (
+	defaultThSeparators  = ""
+	defaultDecSeparators = "."
+	defaultNullValue     = ""
+)
+
+// TypeToTTMapperFactory is a factory capable of creating mappers from Type to Tarantool types.
+type TypeToTTMapperFactory[Type any] interface {
+	// MakeTypeToBooleanMapper creates a mapper from Type to boolean.
+	MakeTypeToBooleanMapper() Mapper[Type, any]
+
+	// MakeTypeToStringMapper creates a mapper from Type to string.
+	MakeTypeToStringMapper() Mapper[Type, any]
+
+	// MakeTypeToUnsignedMapper creates a mapper from Type to unsigned.
+	MakeTypeToUnsignedMapper() Mapper[Type, any]
+
+	// MakeTypeToDatetimeMapper creates a mapper from Type to datetime.
+	MakeTypeToDatetimeMapper() Mapper[Type, any]
+
+	// MakeTypeToUUIDMapper creates a mapper from Type to uuid.
+	MakeTypeToUUIDMapper() Mapper[Type, any]
+
+	// MakeTypeToMapMapper creates a mapper from Type to map.
+	MakeTypeToMapMapper() Mapper[Type, any]
+
+	// MakeTypeToArrayMapper creates a mapper from Type to array.
+	MakeTypeToArrayMapper() Mapper[Type, any]
+
+	// MakeTypeToVarbinaryMapper creates a mapper from Type to varbinary.
+	MakeTypeToVarbinaryMapper() Mapper[Type, any]
+
+	// MakeTypeToDoubleMapper creates a mapper from Type to double.
+	MakeTypeToDoubleMapper() Mapper[Type, any]
+
+	// MakeTypeToDecimalMapper creates a mapper from Type to decimal.
+	MakeTypeToDecimalMapper() Mapper[Type, any]
+
+	// MakeTypeToIntegerMapper creates a mapper from Type to integer.
+	MakeTypeToIntegerMapper() Mapper[Type, any]
+
+	// MakeTypeToNumberMapper creates a mapper from Type to number.
+	MakeTypeToNumberMapper() Mapper[Type, any]
+
+	// MakeTypeToAnyMapper creates a mapper from Type to any.
+	MakeTypeToAnyMapper() Mapper[Type, any]
+
+	// MakeTypeToScalarMapper creates a mapper from Type to scalar.
+	MakeTypeToScalarMapper() Mapper[Type, any]
+
+	// MakeTypeToIntervalMapper creates a mapper from Type to interval.
+	MakeTypeToIntervalMapper() Mapper[Type, any]
+
+	// MakeNullableMapper extends the incoming mapper to a nullable mapper.
+	MakeNullableMapper(Mapper[Type, any]) Mapper[Type, any]
+}
+
+// StringToTTMapperOpts are options of mappers from string to tarantool types.
+type StringToTTMapperOpts struct {
+	// ThSeparators are thousands separators for numeric types.
+	// In fact, these symbols will be ignored for numeric types.
+	// For example:
+	// ThSeparators = "`#e": 1`2`3 -> 123, 123#456#789 -> 123456789, 1e5 -> 15.
+	ThSeparators string
```

Comment:
"th" is not very popular abbreviation. Please use full word - 'Thousand[s]Separators'

Reply:
Changed.